README.md

```diff
@@ -84,35 +84,35 @@ defined('YII_ENV') or define('YII_ENV', 'dev');
 // Alternatives set this in your rr.yaml file
 //defined('YII_ENV') or define('YII_ENV', \getenv('YII_ENV'));
 
-// Load your vendor dependencies and Yii2 autoloader
-require_once '/path/to/vendor/autoload.php';
-require_once '/path/to/vendor/yiisoft/yii2/Yii.php';
+require __DIR__ . '/../vendor/autoload.php';
+require __DIR__ . '/../vendor/yiisoft/yii2/Yii.php';
 
-// Roadrunner relay and PSR7 object
-$relay = new \Spiral\Goridge\StreamRelay(STDIN, STDOUT);
-$psr7 = new \Spiral\RoadRunner\PSR7Client(new \Spiral\RoadRunner\Worker($relay));
+$worker = Spiral\RoadRunner\Worker::create();
+$psrFactory = new Nyholm\Psr7\Factory\Psr17Factory();
+$psr7 = new Spiral\RoadRunner\Http\PSR7Worker($worker, $psrFactory, $psrFactory, $psrFactory);
 
-// Load your configuration file
-$config = require_once '/path/to/config/config.php';
+$config = require __DIR__ . '/../config/web.php';
 
 $application = (new \yii\Psr7\web\Application($config));
 
 // Handle each request in a loop
-while ($request = $psr7->acceptRequest()) {
+while (true) {
     try {
-        $response = $application->handle($request);
-        $psr7->respond($response);
+        $request = $psr7->waitRequest();
+
+        if (!($request instanceof Psr\Http\Message\ServerRequestInterface)) {
+            break;
+        }
+
     } catch (\Throwable $e) {
-        // \yii\Psr7\web\ErrorHandler should handle any exceptions
-        // however you should implement your custom error handler should anything slip past.
-        $psr7->getWorker()->error((string)$e);
+        $psr7->respond(new Nyholm\Psr7\Response(400, [], (string)$e));
     }
 
-    // Workers will steadily grow in memory with each request until PHP memory_limit is reached, resulting in a worker crash.
-    // With RoadRunner, you can tell the worker to shutdown if it approaches 10% of the maximum memory limit, allowing you to achieve better uptime.
-    if ($application->clean()) {
```

Comment:
I haven't kept up with RoadRunner recently. In the past, this was essential to make sure PHP & Yii2s own internal memory leaks didn't catch up and cause a crash. Has this been resolved? If not we should add it back in instead of removing it.

Reply:
Well I have been using RoadRunner for exactly 3 days, I have no much expertise. I think I will be wise to be extra cautious bringing back the piece of code you proposed initially.